packages/components/pie-modal/src/modal.scss

```diff
@@ -21,6 +23,10 @@
 
     box-shadow: var(--modal-elevation);
 
+    &::backdrop {
+        background: #{dt.$color-overlay};
```

Comment:
I'd potentially put a comment here about why we're using Sass variables for backdrop.

Reply:
Fixed.

Reply:
I'd probably expand on this comment. It's a bit confusing because we know we can declare variables with the scope of `::backdrop`. Is this caused by a top layer/shadow DOM issue? 

Just mentioning it because at first glance I'm not sure this comment makes sense to me